### PR TITLE
Create oxford-iiit-pets-segmentation-using-pytorch.ipynb

### PR DESCRIPTION
**Pull request by dhruvbird**
Created a notebook using a PyTorch model (SegNet) for IIIT Oxford Pets Segmentation.

### CODE REVIEW

**Comment by reddragon** [comment]
Can you share a Google Colab link? I can't view the file directly in GitHub and it doesn't let me download either.

**Comment by NareshPS** [comment]
This file is 31 MB. Can you trim it down please?

**Comment by dhruvbird** [comment]
> This file is 31 MB. Can you trim it down please?

Reduced to about 5 MiB.

**Comment by reddragon** [comment]
Why are the predicted labels are all dark in the latest version on Kaggle?

**Comment by dhruvbird** [comment]
@reddragon updated the notebook to include the segmentation masks after 20 training epochs for both models.
